packages/atoms/src/components/kiwi-dropdown/kiwi-dropdown.tsx

```diff
@@ -72,6 +76,29 @@ export class KiwiDropdown implements ComponentInterface {
   private handleMenuEnter: () => void = () => (this.hover = true);
   private handleMenuLeave: () => void = () => (this.hover = false);
 
+  componentDidRender(): void {
+    // make sure that the dropdown does not stick over the right edge of the body
+    if(this.el) {
+      const dropdown = this.el.querySelector('.dropdown-menu') as HTMLElement;
```

Comment:
Anstatt `@Element()` und dann darin zu suchen, kannst du gleich an dem `div.dropdown-menu` das `ref`-Attribut setzen. Siehe https://stenciljs.com/docs/templating-jsx#getting-a-reference-to-a-dom-element

Das geht natürlich in anderen Fällen nicht, wo das gewünschte Element nicht in der render-Methode derselben Komponente auftaucht.